Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -323,6 +323,8 @@ void AVIDump::CloseVideoFile()
   av_frame_free(&s_scaled_frame);
 
   avcodec_free_context(&s_codec_context);
+
+  avio_closep(&s_format_context->pb);
```

Comment:
Possibly a dumb question (I know little about this code), is it possible for s_format_context to be null?